### ISSUE TITLE
Modify notifications for the pre-check-less case scenario

### ISSUE THREAD

**Issue by mkaczanowski** [ChorusOne/blazar#28]
# TL;DR
In the current code, if the pre-checks are skipped and the upgrade is executed via `upgrade-info.json` path, [there are no (or just one) slack notifications](https://github.com/ChorusOne/blazar/blob/6c749754e97b7b6a2651d8cc395647315b1fcda2/internal/pkg/daemon/daemon.go#L437).

That is confusing to network operator as he doesn't see the upgrade happening.

It would be great to add slack notifications to `performUpgrade` so that:
1. we know at the entry the upgrade code path has been triggered
2. major steps of the upgrade have notifications (e.g when the version is replaced in the env file etc)
